pretrain_gpt.py

```diff
@@ -217,10 +233,11 @@ def forward_step(data_iterator, model, teacher_model=None):
     moe_loss = sum(moe_losses) * args.moe_loss_coeff
 
     mos_loss = 0
-    if args.mos:
+    if args.mos or args.kd:
         assert model.training
         mos_loss = calculate_mos_loss(args, stu_output, teacher_model, tokens, position_ids, attention_mask)
     
+    print_rank_0('*** args.kd {}, kd loss {}'.format(args.kd, mos_loss))
```

Comment:
This seems to be a debug logging and no longer needed? It will print under non-kd cases, and there is a "print_rank_0('>>> total loss: {}, lm loss {}, kd loss {}'.format(loss, averaged_loss[0], mos_loss))" above in the same file.